ts/screens/wallet/TransactionsScreen.tsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * This screen dispalys a list of transactions
+ * from a specific credit card
+ */
+import * as React from "react";
+import I18n from "../../i18n";
+
+import { Content, View } from "native-base";
+import { Image } from "react-native";
+import {
+  NavigationInjectedProps,
+  NavigationScreenProp,
+  NavigationState
+} from "react-navigation";
+
+import { WalletAPI } from "../../api/wallet/wallet-api";
+import { WalletStyles } from "../../components/styles/wallet";
+import { WalletLayout } from "../../components/wallet/layout/WalletLayout";
+import { TransactionsList } from "../../components/wallet/TransactionsList";
+import { CreditCard } from "../../types/CreditCard";
+import { WalletTransaction } from "../../types/wallet";
+
+import { topContentTouchable } from "../../components/wallet/layout/types";
+
+const cardsImage = require("../../../img/wallet/card-tab.png");
+
+interface ParamType {
+  readonly card: CreditCard;
+}
+
+interface StateParams extends NavigationState {
+  readonly params: ParamType;
+}
+
+interface OwnProps {
+  readonly navigation: NavigationScreenProp<StateParams>;
+}
+
+type Props = OwnProps & NavigationInjectedProps;
+
+export class TransactionsScreen extends React.Component<Props, never> {
+  constructor(props: Props) {
```

Comment:
Is this usefull?

Reply:
I would daresay it is not. Getting rid of those